Porting/Maintainers.pl

```diff
@@ -677,10 +677,6 @@ package Maintainers;
         'EXCLUDED'     => [
             qr{^examples/},
         ],
-        'CUSTOMIZED' => [
```

Comment:
The entry in `t/porting/customized.dat` also needs to be removed.

Reply:
Hmm ... that suggests that `make test_porting` failed to detect the need to remove that entry.

And when I ran `cd t; ./perl -I../lib porting/customized.t --regen; cd -`, I discovered that `make test_porting` had similarly failed to report the need to modify the database after d91c117aa9f last night.

Reply:
@tonycoz, customized.dat has been updated; can you re-review? Thanks.